omnibus/config/software/datadog-agent.rb

```diff
@@ -102,7 +102,13 @@
       move 'bin/agent/dist/system-probe.yaml', "#{conf_dir}/system-probe.yaml.example"
   end
   move 'bin/agent/dist/conf.d', "#{conf_dir}/"
-  copy 'bin/agent', "#{install_dir}/bin/"
+
+  unless windows?
+    copy 'bin/agent', "#{install_dir}/bin/"
+  else
+    copy 'bin/agent/ddtray.exe', "#{install_dir}/bin/agent"
+    copy 'bin/agent/dist', "#{install_dir}/bin/agent"
```

Comment:
Instead of having to specify the files we want, could we instead specify the files to exclude (ie: `*.pdb`)? Otherwise, if we ever add a new binary or directory there, it will be really hard to find why it doesn't show up on Windows.

Reply:
In this case, however, we do copy everything on non-Windows platforms. I can imagine someone developing a feature on Linux/Mac without having to change this recipe and then being surprised when it doesn't work on Windows.